Multisig/Features/Connect to Web/Logic/WebConnectionController.swift

```diff
@@ -655,6 +658,40 @@ class WebConnectionController: ServerDelegateV2, RequestHandler, WebConnectionSu
         notifyObservers(of: request)
     }
 
+    func prepareTransactionForExecution(_ request: WebConnectionSendTransactionRequest) {
+        guard
+            let connection = connection(for: request),
+            let chainId = connection.chainId,
+            let chain = chain(for: request),
+            let features = chain.features
+        else {
+            return
+        }
+
+        let minerTip: Sol.UInt256 = 1_500_000_000
+
+        // select tx type based on the chain.
+        let isEIP1559 = features.contains("EIP1559")
+        if isEIP1559 {
+            request.transaction = Eth.TransactionEip1559(
+                chainId: Sol.UInt256(chainId),
+                from: request.transaction.from,
+                to: request.transaction.to,
+                value: request.transaction.value,
+                input: request.transaction.data,
+                fee: .init(maxPriorityFee: minerTip)
+            )
+        } else {
+            request.transaction = Eth.TransactionLegacy(
```

Comment:
would EIP-2930 type be handled here correctly?

Reply:
Do we need to handle EIP-2930? We're not using access lists. Do we allow the creation of a tx with EIP-2930 specific featueres? For example access lists?